test/duckdb_test.rb

```diff
@@ -5,5 +5,9 @@ class DuckDBTest < Minitest::Test
     def test_that_it_has_a_version_number
       refute_nil ::DuckDB::VERSION
     end
+
+    def test_that_it_has_a_library_version_number
+      refute_nil ::DuckDB::LIBRARY_VERSION if defined? DuckDB.library_version
```

Comment:
Can we use `skip` if `DuckDB.library_version` is not available?
https://github.com/minitest/minitest#unit-tests-